src/service/stockService.js

```diff
@@ -44,5 +44,18 @@ export default class StockService {
 
         return stockWithUsers;
     }
+
+    async postCandlestickData(stockData) {
+        try {
+            var ticker = await this.stockRepository.getStockTicker(stockData.Ticker)
+            if (undefined == ticker) {
```

Comment:
you need to explain to me why this is better. I still don't get it.

Reply:
Better than what?